osu.Game/Screens/Play/HUD/SpectatorList.cs

```diff
@@ -232,11 +236,17 @@ private void updateEnabledState()
                     linkCompiler.Enabled.Value = UserPlayingState.Value != LocalUserPlayingState.Playing;
             }
         }
+    }
+
+    public partial class SkinnableSpectatorList : SpectatorList, ISerialisableDrawable
+    {
+        public bool UsesFixedAnchor { get; set; }
 
-        public record Spectator(int OnlineID, string Username) : IUser
+        [BackgroundDependencyLoader]
+        private void load(SpectatorClient client, GameplayState gameplayState)
         {
-            public CountryCode CountryCode => CountryCode.Unknown;
-            public bool IsBot => false;
+            ((IBindableList<SpectatorUser>)Spectators).BindTo(client.WatchingUsers);
+            ((IBindable<LocalUserPlayingState>)UserPlayingState).BindTo(gameplayState.PlayingState);
```

Comment:
I find it super odd that these bindings are done here. I get that this is to allow the base class to be tested, but it doesn't sit well. Two proposals:

- Rename this class to `OnlineSpectatorList` (dunno)
- Change how the testing is done to not require that and move the `ISerialisableDrawable` to the base implementation

Leaning towards the second.